tools/spm_decode

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
what's the benefits of wrapping spm_encode/spm_decode/spm_train? 
If just for easier input output interface, the raw spm_* are recommended.